src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -913,6 +913,9 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>The package reference {0} specifies a VersionOverride but the ability to override a centrally defined version is currently disabled.</value>
     <comment>0 - The name of a package reference.</comment>
   </data>
+  <data name="Warning_CentralPackageVersions_MultipleSourcesWithoutPackageSourceMapping" xml:space="preserve">
+    <value>There are more than one package sources configured but package source mapping is not configured.  To use central package management, please configure package source mapping or use a single package source.</value>
```

Comment:
There are {0} package sources defined in your configuration. To enable central package management, please map your package sources with {aka.ms link} or specify a single package source to continue.

{0} - Number of sources.

{aka.ms link} - https://docs.microsoft.com/en-us/nuget/consume-packages/package-source-mapping

@nkolev92 thoughts?

Reply:
Love it.

Reply:
@nkolev92  do we already have an aka.ms link to package source mapping or should I just embed the URL in the message?

Reply:
I can create a link as I synced with nikolche and we use aka.ms for a number of NuGet and sdk things. What alias do we want @nkolev92?

nugetsourcemapping?
packagesourcemapping?
etc

Reply:
either one works. Is there a pattern we have with some of our other aka.ms links?

Reply:
Gentle ping, I'd like to merge this today.  I can merge it for now with the link directly to the docs or we need to decide on the aka.ms link and I can use that even if it doesn't exist yet

Reply:
https://aka.ms/nuget-package-source-mapping

Reply:
Or if you want to use the docs link, that's fine too. Just remove the locale `en-us`. if you decide that way.

Reply:
Just saw it come through, I've updated the PR to have https://aka.ms/nuget-package-source-mapping